server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -919,7 +906,7 @@ public Builder version(long version) {
         }
 
         public Builder columnOID(long columnOID) {
```

Comment:
This is deprecated, right?
```suggestion
        @Deprecated
        public Builder columnOID(long columnOID) {
```